content/en/docs/components/hyperparameter-tuning/experiment.md

```diff
@@ -181,7 +181,8 @@ sampling without replacement. Random search is therefore the best algorithm to
 use when combinatorial exploration is not possible. If the number of continuous
 variables is high, you should use quasi random sampling instead.
 
-Katib uses the [hyperopt](http://hyperopt.github.io/hyperopt/) optimization
+Katib uses the [hyperopt](http://hyperopt.github.io/hyperopt/) or
```

Comment:
I have no good idea how describe `chocolate-random` and `chocolate-quasirandom`. So I added Goptuna  only.

It may be better to rename `chocolate-random` to `random` for consistency (related discussion: https://github.com/kubeflow/katib/pull/1131#discussion_r407955790).

Reply:
I agree with @c-bata. What do you think @gaocegege @johnugeorge ?

Reply:
SGTM!